element_deeplabcut/readers/dlc_reader.py

```diff
@@ -33,33 +33,38 @@ def __init__(
             self.dlc_dir = Path(dlc_dir)
             assert self.dlc_dir.exists(), f"Unable to find {dlc_dir}"
 
-        # meta file: pkl - info about this  DLC run (input video, configuration, etc.)
+        # meta file: pkl - info about this DLC run (input video, configuration, etc.)
         if pkl_path is None:
-            pkl_paths = list(self.dlc_dir.rglob(f"{filename_prefix}*meta.pickle"))
-            assert len(pkl_paths) == 1, (
-                "Unable to find one unique .pickle file in: "
-                + f"{dlc_dir} - Found: {len(pkl_paths)}"
+            self.pkl_paths = sorted(
+                self.dlc_dir.rglob(f"{filename_prefix}*meta.pickle")
             )
-            self.pkl_path = pkl_paths[0]
+            assert (
+                len(self.pkl_paths) > 0
+            ), f"No meta file (.pickle) found in: {self.dlc_dir}"
         else:
-            self.pkl_path = Path(pkl_path)
-            assert self.pkl_path.exists()
+            pkl_path = Path(pkl_path)
+            assert pkl_path.exists()
+            self.pkl_paths = [pkl_path]
 
         # data file: h5 - body part outputs from the DLC post estimation step
         if h5_path is None:
-            h5_paths = list(self.dlc_dir.rglob(f"{filename_prefix}*.h5"))
-            assert len(h5_paths) == 1, (
-                "Unable to find one unique .h5 file in: "
-                + f"{dlc_dir} - Found: {len(h5_paths)}"
-            )
-            self.h5_path = h5_paths[0]
+            self.h5_paths = sorted(self.dlc_dir.rglob(f"{filename_prefix}*.h5"))
+            assert (
+                len(self.h5_paths) > 1
```

Comment:
Do you want more than one here?
```suggestion
                len(self.h5_paths) > 0
```

Reply:
`> 0` is what I meant, thanks for the catch

Reply:
fixed